src/Tooltip/Tooltip.js

```diff
@@ -152,7 +169,8 @@ Tooltip.defaultProps = {
   onMouseEnter: undefined,
   onMouseLeave: undefined,
   onOpen: undefined,
-  style: {}
+  style: {},
+  direction: 'down'
```

Comment:
by default make the tooltip display above an element (so make it `position: 'top'`